themes/socialblue/src/Plugin/Preprocess/Page.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Drupal\socialblue\Plugin\Preprocess;
+
+use Drupal\socialbase\Plugin\Preprocess\Page as PageBase;
```

Comment:
We should check if we should overwrite the Socialbase Page Preprocessor here because I think they are all executed by Bootstrap, this would case the SocialBase one to be executed twice. Use the Bootstrap PreprocessBase instead.

Reply:
In this case, we need to use `$variables['content_attributes']`, which was defined only in socialbase, but not exists in bootstrap. It gives error when I try to extend it directly from bootstrap, so we need to leave it like this.